### DISCUSSION TITLE
Suggestion: Gravity!

### DISCUSSION THREAD

**Discussion by Phrenotopia** [MaxRobinsonTheGreat/EvolutionSimulatorV2#97] [Ideas and Suggestions]
This will probably take an enormous toll on performance, and this may be better off in a more advanced branch of this project. 

I just thought it would be a neat idea to add some kind of physics like gravity to begin with. I presume the most straightforward way (but costly performance-wise) would be to iterate all non-living pixels as well as all organisms for a given amount of ticks, and move them one position down. Perhaps there are smarter ways to do this, like e.g. a physics engine.

**Comment by doo158** [comment]
there would probably need to be some different cell types for this to work, like a cell that allows creatures to climb
